experimental/PlaneCurve/docs/doc.main

```diff
@@ -1,3 +1,9 @@
 [
-   "plane_curves.md",
+    "Plane Curves" => [
+        "introduction.md",
+        "plane_curves.md",
+        "divisors.md",
+        "elliptic_curves.md",
+        "non_plane_curves.md",
+    ],
 ]
```

Comment:
If this should be disabled, the easiest way should be to remove `doc.main` or maybe rename it to something like `doc.main.disabled`?

Reply:
Hmm, is that easier than commenting out the code here? Commenting out also means that it is easier to "rediscover" these docs and to notice that they are deactivated; and by adding a comment we can even explain why...

Reply:
If commenting out works. It should though, since I only copied the other code for reading the `doc.main` from elsewhere.